notebooks/concepts/SpecViz.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
Two things about this notebook - I think it should be in `notebooks`, not `notebook/concepts` - concepts is for notebooks that don't actually work, and this is probably a real-live example.

Second, the notebook should have its outputs cleared - it's best to only commit output-cleared notebooks in github so as not to fill up the repo with outputs, and to not confuse users when they try to run it for the first time.  Normally I'd say you should also go back and clean it from commit history but it's not that big a deal if there's no images.